.evergreen/download-mongodb.sh

```diff
@@ -699,13 +699,9 @@ download_and_extract_package ()
    MONGODB_DOWNLOAD_URL=$1
    EXTRACT=$2
 
-   if [ -n "$MONGODB_BINARY_ROOT" ]; then
-      cd $MONGODB_BINARY_ROOT
-   else
-      SCRIPT_DIR=$(dirname ${BASH_SOURCE:-$0})
-      . $SCRIPT_DIR/handle-paths.sh
-      cd $DRIVERS_TOOLS
-   fi
+   cd $(dirname $(dirname ${MONGODB_BINARIES}))
```

Comment:
```suggestion
   cd $(dirname $(dirname ${MONGODB_BINARIES:?}))
```
Given `dirname` requires an argument, `MONGODB_BINARIES` being defined prior to this function is a precondition which should be asserted. Use `${var:?}` syntax to assert `MONGODB_BINARIES` is not null or empty. Otherwise, ensure there is a proper fallback strategy (e.g. using `MONGODB_BINARY_ROOT`?) if the variable is null or empty.